legion/jupyterlab-plugin/jupyter_legion/handlers/cloud.py

```diff
@@ -0,0 +1,252 @@
+#
+#    Copyright 2019 EPAM Systems
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
```

Comment:
Keep just vcs name. 
![Screenshot from 2019-05-29 21-04-11](https://user-images.githubusercontent.com/6774206/58576423-5f0c8d80-8255-11e9-95b4-f9b18421d90f.png)

Reply:
I guess that just name is not too informative as name + remote + def branch

Reply:
It looks awful.
We can create a separate view for vcs with all parameters.
Anyway I can't see the branch on the screen.

Reply:
Removed